### PR TITLE
Add thumbnails configuration page

### PR DESCRIPTION
**Pull request by PhillippOhlandt**
Preparation for adding documentation about thumbnail aliases in 3.2.

Feel free to suggest text changes :)

### CODE REVIEW

File: docs/configuration/thumbnails.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    default_thumbnail: [ 160, 120 ]
+    default_image: [ 1000, 750 ]
+    quality: 80
+    cropping: crop
+    notfound_image: view/img/default_notfound.png
+    error_image: view/img/default_error.png
+    save_files: false
+    allow_upscale: false
+    exif_orientation: true
+#    browser_cache_time: 2592000
```
Comment (GawainLynch) [inline docs/configuration/thumbnails.md]: Question to the audience: Do we still use `browser_cache_time`

Comment (PhillippOhlandt) [inline docs/configuration/thumbnails.md]: I took it from the configuration :D

**Comment by PhillippOhlandt** [comment]
@bobdenotter your turn :)

File: docs/configuration/thumbnails.md (hunk 1 above)
Comment (bobdenotter) [inline docs/configuration/thumbnails.md]: I don't think it's used anymore: 

```
$ ag 'browser_cache_time' src                                 
src/Provider/ThumbnailsServiceProvider.php
69:        $app['thumbnails.cache_time'] = $app['config']->get('general/thumbnails/browser_cache_time');

$ ag 'thumbnails.cache_time' vendor/bolt/thumbs         
vendor/bolt/thumbs/src/ServiceProvider.php
31:                $app['thumbnails.cache_time']
53:        $app['thumbnails.cache_time'] = null;
```

Should we just remove it? (both here, and in the source code)

**Comment by bobdenotter** [comment]
I really like he tables. We should definitely use more of them! :-)

Only thing is, they need to be linearized, because currently they break the layout on mobile: 

screen shot 2016-11-01 at 07 06 44

Perhaps "stack" them on smaller screens? http://foundation.zurb.com/sites/docs/table.html#stacked-table

File: docs/configuration/thumbnails.md (hunk 1 above)
Comment (bobdenotter) [inline docs/configuration/thumbnails.md]: If it's only _set_ and then nothing is done with it, does that count as "being in use"? Or am I missing something? 

(not saying we _should_ remove it, just curious)

Comment (GawainLynch) [inline docs/configuration/thumbnails.md]: The service provider [passes that value](https://github.com/bolt/bolt-thumbs/blob/master/src/ServiceProvider.php#L31) into the [Responder](https://github.com/bolt/bolt-thumbs/blob/master/src/Responder.php#L56) class and then uses it as a parameter to the [caching call](https://github.com/bolt/bolt-thumbs/blob/master/src/Responder.php#L97).

Comment (bobdenotter) [inline docs/configuration/thumbnails.md]: Ah, duh!

Comment (GawainLynch) [inline docs/configuration/thumbnails.md]: Yeah, I has a similar "duh" moment around my original question … was just in too much of a hurry yesterday to do 2 seconds worth of work, that cost you and I far more than that in the end … 

Lesson to be learned there, Gawain? :wink:

**Comment by GawainLynch** [comment]
FWIW :+1: from me

**Comment by PhillippOhlandt** [comment]
Ok, changing the parameters to be `code`. I had that before, It just looked less clean.

**Comment by bobdenotter** [comment]
Looks good to me! 

🎏

**Pull request merged**